### PR TITLE
Submit extrinsic

### PR DESCRIPTION
**Pull request by philipstanislaus**

### CODE REVIEW

File: rpc/author/author.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import (
+	"github.com/centrifuge/go-substrate-rpc-client/client"
+)
+
+type Author struct {
+	client client.Client
+}
+
+func NewAuthor(cl client.Client) *Author {
```
Comment (vedhavyas) [inline rpc/author/author.go]: Comments pls

Comment (philipstanislaus) [reply rpc/author/author.go]: Sure, added

File: rpc/author/submit_extrinsic.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+import (
+	"github.com/centrifuge/go-substrate-rpc-client/types"
+)
+
+// SubmitExtrinsic will submit a fully formatted extrinsic for block inclusion
+func (a *Author) SubmitExtrinsic(xt types.Extrinsic) (types.Hash, error) {
+	enc, err := types.EncodeToHexString(xt)
+	if err != nil {
+		panic(err)
```
Comment (vimukthi-git) [inline rpc/author/submit_extrinsic.go]: a library must not panic.

Comment (philipstanislaus) [reply rpc/author/submit_extrinsic.go]: Changed on `master`

**Pull request merged**